src/backend/distributed/sql/downgrades/citus--11.1-1--11.0-2.sql

```diff
@@ -46,4 +46,19 @@ CREATE FUNCTION pg_catalog.worker_repartition_cleanup(bigint)
  STRICT
 AS 'MODULE_PATHNAME', $function$worker_repartition_cleanup$function$;
 
-#include "../../../columnar/sql/downgrades/columnar--11.1-1--11.0-2.sql"
+-- add relations to citus
+ALTER EXTENSION citus ADD SCHEMA columnar;
```

Comment:
I tried downgrading to 10.2 and 11.0, but in both cases got:
```
postgres=# alter extension citus update;
ERROR:  schema columnar is already a member of extension "citus_columnar"
```

Reply:
Hmz, still get the same issue. My steps are:

1. Install this branch, create a columnar table
2. Install release-11.0 branch, restart
3. `make install-downgrades` on this branch
4. `alter extension citus update;`

Reply:
sorry I am not quite understand, step 1 install citus extension and create a columnar table, on my branch the installed version will be 11.1-1 for both citus & citus_columnar, then step 2 checkout release-11.0 branch, rerun make install & make xx, and restart the server ? step 4: in this case the citus already being installed and still trying to run "alter extension citus update" ?

Reply:
I think for those steps, since you re-install the release-11.0 branch, the utility_hook.c is an old version that do not contain my changes to drop the relations from citus_columnar first (citus_columnar--11.1-1--11.1-0.sql ), that's why it will throw the errors that "schema columnar is already a member of extension "citus_columnar""

Reply:
I think with this change, we're setting the assumption that downgrades run ALTER EXTENSION on the binaries of the new version with `set citus.enable_version_checks to off`, before installing the old version and restarting. 

I think that's ok, since that's the only path that is at least somewhat tested, but I do have one question remaining: Why can we not have a regular downgrade script for this case?

Reply:
I was using regular sql script for upgrade and downgrade initially, but that requires lots of conditional checks and the scripts execution order is kind of out of our control e.g during upgrades, we need citus_columnar to be installed first etc, those could not be done just through sql scripts so we turn to use utility_hook for all cases eventually